README.md

```diff
@@ -1,19 +1,15 @@
 # npm-ui
```

Comment:
name should also be changed here

Reply:
There's a bunch of places, thought I'd see if people like the name first.